scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -3141,6 +3141,10 @@ def forward_pass_and_gradients():
         # All-reduce gradients using Horovod.
         grads = [hvd.allreduce(grad, average=False, device_dense=horovod_device)
                  for grad in grads]
+      elif self.params.variable_update == 'kungfu':
```

Comment:
Is it possible that we create a new benchmark.py to contain kungfu related code? If we keep the tf_cnn_benchmark untouched, it would be easy for us to merge with the upstream.

Reply:
I rethink about this. Let us keep in such a way to enable kungfu right now. We can create the benchrmark.py later.